### PR TITLE
Refactor users and ranger policies variables with tdp collection playbooks

### LINKED ISSUE #180: Refactor users and ranger policies variables with tdp collection playbooks
In tdp-collection, these PR added new playbooks for created HDFS home dirs and Ranger policies:
- https://github.com/TOSIT-IO/tdp-collection/pull/438
- https://github.com/TOSIT-IO/tdp-collection/pull/444

Refactor getting started to use it.

### PR DESCRIPTION
**Pull request by rpignolet**
#### Which issue(s) this PR fixes:

Fixes #180 

#### Additional comments:

To make clear that you license your contribution under
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0) and that you give permission to [TOSIT](https://www.tosit.io/),
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

**Review by mehdibn — APPROVED** [review]

**Comment by rpignolet** [comment]
Last force push update `tdp-collection` git submodule to have the new utils playbooks used.

**Review by gboutry — APPROVED** [review]

**Comment by rpignolet** [comment]
Last for push adding documentation about restarting Ranger Usersync when adding new users and remove `deploy-users.yml` playbook.

File: README.md [hunk 1]
```diff
@@ -426,20 +428,33 @@ kafka-console-consumer.sh --topic test-topic --from-beginning \
...
 
 ```
-ansible-playbook deploy-users.yml
+ansible-playbook ansible_roles/collections/ansible_collections/tosit/tdp/playbooks/utils/hdfs_user_homes.yml
 ```
 
-_Additional users can be added to the Ansible playbook parameter `users` in the `deploy-users.yml` if required._
+_Additional users can be added to the Ansible variable `hdfs_user_homes` if required._
+
+When adding users after Ranger Usersync deployment and want to add or update Ranger policies with these new users, you must wait until Ranger Usersync poll users from LDAP or restart the Ranger Usersync with the playbook:
```
Comment (gboutry) [inline README.md:443]:
```suggestion

When adding users following the Ranger Usersync deployment, you will need to add or update Ranger policies including these new users. You must wait for Ranger Usersync to poll users from LDAP or you can restart the Ranger Usersync using the following playbook:
```

**Review by gboutry — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rpignolet) [reply README.md]: Done in last force push

**Review by gboutry — APPROVED** [review]

**Pull request merged**